packages/web/src/press/PressPage.tsx

```diff
@@ -26,17 +26,20 @@ interface Props {
 class PressPage extends React.PureComponent<I18nProps & Props> {
   static async getInitialProps(context) {
     let press = []
-    let languages
+    let languages = []
     try {
       // when run on server import fetching code and run. on client send req to api
       if (context.req) {
         const getpress = await import('src/../server/fetchPress')
         press = await getpress.default()
         languages = context.req.headers['accept-language']
+          .toLowerCase()
+          .split(',')
+          .map((s) => s.substr(0, 2))
       } else {
         const res = await fetch(`/api/press`)
         press = await res.json()
-        languages = res.headers.get('accept-language')
+        languages = navigator.languages.map((s) => s.substr(0, 2))
```

Comment:
because `navigator.languages` doesnt have 100% support can you make so even if its undefined this wont break? in that case our languages array would just be empty.

Reply:
ok! I set it to an empty list if it's undefined, and then on line 56, it'll just use the i18n language instead